Automattic/jetpack
# Eventbrite Block: Update modal embed preview

Fixes #14466

#### Changes proposed in this Pull Request:

* Alternative to https://github.com/Automattic/jetpack/pull/14469 that maintains a dynamic block preview for the modal embed type.

#### Testing instructions:

- On a Jetpack Docker environment, run `yarn docker:wp core update --version=5.2 --force` to downgrade to WP 5.2.
- Open the editor, insert an Eventbrite block, and add an Eventbrite event URL.
- Check in the block sidebar that the "Embed type" section looks ok at all screen sizes.
- Save and reload: make sure the block show up and works as expected.


#### Proposed changelog entry for your changes:

* None needed
